components/arm/xarm/xarm.go

```diff
@@ -255,5 +255,25 @@ func (x *xArm) DoCommand(ctx context.Context, cmd map[string]interface{}) (map[s
 	if _, ok := cmd["load"]; ok {
 		return x.getLoad(ctx)
 	}
+	if val, ok := cmd["set"]; ok {
+		// The command expects a float64 slice of length 2, describing the desired speed and acceleration.
+		// We specify our values in degrees and here they are converted into radians.
+		interfaceList, err := utils.AssertType[[]interface{}](val)
```

Comment:
Rather than forcing setting of speed and acceleration together as a list, let's add "set_speed" and "set_acceleration" as separate commands.